SPEC.md

```diff
@@ -85,33 +85,54 @@ This specification outlines the requirements for botbuilder samples.
 ```markdown
 <INSERT AT MOST ONE PARAGRAPH DESCRIPTION OF WHAT THIS SAMPLE DOES> 
 
+# Table of Contents
+<LINKS TO SECTIONS OF THE README>
+
 # Concepts introduced in this sample
 <DESCRIPTION OF THE CONCEPTS>
 
+#Services used in this sample
+<INTRODUCE SERVICES>
```

Comment:
This should really be <LIST SERVICES USED> instead of <INTRODUCE SERVICES> - right? Keep this light weight and point to services instead of describing what the services do (which should be in the concepts introduced section)

Reply:
Yes i was thinking this could basically just be something like "LUIS - <LINK TO LUIS>" then maybe a sentence or so like "LUIS does NLP..."